src/__tests__/test.js

```diff
@@ -7,7 +7,9 @@ describe('Simple Linear Regression', () => {
 
     const regression = new SLR(inputs, outputs);
 
+    // @ts-expect-error coefficients possibly undefined
     expect(regression.slope).toBe(regression.coefficients[1]);
+    // @ts-expect-error coefficients[1] possibly undefined
     expect(regression.intercept).toBe(regression.coefficients[0]);
```

Comment:
In what case can coefficients be undefined?

Reply:
I think this is in the case that `coefficients` key doesn't exist

Reply:
I've done the typing on an extra branch, and the test comments can be removed in the next PR